Keymaps/Default (Linux).sublime-keymap

```diff
@@ -335,4 +335,16 @@
     //         }
     //     ]
     // },
+    // Internal key-binding
+    {
+        "keys": [
+            "<character>"
+        ],
```

Comment:
It feels dangerous, but let's try it anyway. Note that things like backspace and the enter key are not handled by `<character>`. But I think there's a low chance that anyone is pressing those keys.